### DISCUSSION TITLE
Add sensitivity to osu!catch.

### DISCUSSION THREAD

**Discussion by MilkCat03** [ppy/osu#17000] [Ideas]
The sensitivity could be how fast the catcher moves. This will help a lot of pc players switch to laptop. This will also fix the current issue of the sensitivity being too high.

**Comment by bdach** [comment]
Would this not make the game substantially easier / harder depending on map?

This may make sense but as a mod. Definitely don't think arbitrary adjustments that don't affect score submission in *some* way are a good idea.

**Reply by peppy** [reply]
As a mod sounds pretty good.

**Reply by TheFantasticLoki** [reply]
> As a mod sounds pretty good.

Could this be implemented into difficulty adjust specifically since that basically fits the description? No real need for it to have it's own mod.

**Reply by iSimpFor** [reply]
i feel like it could be a beatmap setting, where all beatmaps before the feature all have a certain sensitivity

sensitivity could then be multiplied with inherited lines

then the sensitivity can be edited via difficulty adjust mod

**Reply by frenzibyte** [reply]
Unsure about going straight there instead of starting with a mod in the "Fun" category.

**Reply by iSimpFor** [reply]
well it's a idea of what the idea could build up to, but we can start with a mod though i think hyperdashes need to be applied based on the sensitivity
